core/src/main/java/io/questdb/griffin/ExpressionParser.java

```diff
@@ -849,6 +853,9 @@ void parseExpr(GenericLexer lexer, ExpressionParserListener listener, SqlParserC
                                     }
                                 }
 
+                                // there is one case for valid dangling expression - when we create alias for column (like 'value' 'x' equivalent to 'value' as 'x')
```

Comment:
nit: like is a SQL keyword
```suggestion
                                // there is one case for valid dangling expression - when we create an alias for column (`'value' 'x'` equivalent to `'value' as 'x'`)
```